core/src/main/java/io/kestra/core/http/client/apache/FailedResponseInterceptor.java

```diff
@@ -12,22 +12,43 @@
 
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
+import java.util.List;
 
-@AllArgsConstructor
 public class FailedResponseInterceptor implements HttpResponseInterceptor {
+    private Boolean allErrors = false;
```

Comment:
`allErrors` is is setted in the two constructors so `private final boolean allErrors`